backend/config.js

```diff
@@ -113,4 +121,6 @@ module.exports = {
       },
     },
   },
+
+  phaseName: phaseName(),
```

Comment:
I'd just expose the name without checks. 
`phaseName: process.env.SYSTEM_PHASE`